src/main/java/seedu/address/MainApp.java

```diff
@@ -61,8 +72,20 @@ public void init() throws Exception {
 
         initLogging(config);
 
+        /*
+        Step 2. Create game here.
+
+        game = initGameManager(storage, userPrefs);
```

Comment:
Should we initialize the game object upon immediately starting the app? 
Perhaps we can consider only initializing when the user types the 'start' command!

Reply:
Great thought. We would have to work out deliverables of Game package to know if we require certain functionalities before the game starts.

I was thinking of this use case: users get to see past game attempts and resume a possibly paused attempt when they open the app. Game manager could serve some of this functionalities.